code/modules/mob/living/living_movement.dm

```diff
@@ -12,3 +12,6 @@
 		remove_movespeed_modifier(MOVESPEED_ID_BULKY_DRAGGING)
 		return
 	add_movespeed_modifier(MOVESPEED_ID_BULKY_DRAGGING, TRUE, 0, NONE, TRUE, .)
+
+/mob/living/set_submerge_level(turf/new_loc, turf/old_loc, submerge_icon = 'icons/turf/alpha_64.dmi', submerge_icon_state = "liquid_alpha", duration = cached_multiplicative_slowdown + next_move_slowdown)
```

Comment:
Having what's basically assignment logic in the proc declaration is weird but I guess it works?

Reply:
I already have the check in the base proc, so didn't see any value in requiring more copy paste checks for every version of the proc.